README.md

```diff
@@ -293,3 +293,96 @@ environments:
 2017-08-10 17:21:58 bucket4
 2017-08-15 20:33:02 bucket5
 ```
+
+### `fleece config`
+
+```
+usage: fleece config [-h] [--config CONFIG] [--username USERNAME]
+                     [--apikey APIKEY] [--environments ENVIRONMENTS]
+                     {import,export,edit,render,upload} ...
+
+Configuration management
+
+positional arguments:
+  {import,export,edit,render,upload}
+                        Sub-command help
+    import              Import configuration from stdin
+    export              Export configuration to stdout
+    edit                Edit configuration
+    render              Render configuration for a stage
+    upload              Upload configuration to SSM
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --config CONFIG, -c CONFIG
+                        Config file (default is config.yml)
+  --username USERNAME, -u USERNAME
+                        Rackspace username. Can also be set via RS_USERNAME
+                        environment variable
+  --apikey APIKEY, -k APIKEY
+                        Rackspace API key. Can also be set via RS_API_KEY
+                        environment variable
+  --environments ENVIRONMENTS, -e ENVIRONMENTS
+                        Path to YAML config file with defined accounts and
+                        stage names. Defaults to ./environments.yml
+```
+
+The `fleece config` command has a few sub-commands that work with configuration files. There are a number of arguments that apply to all commands:
+
+- `--config` sets the configuration file. This is the file that holds the configuration, in a format that is appropriate to commit to source control (i.e. sensitive variables are encrypted)
+- `--username` and `--apikey` are the Rackspace credentials, used to obtain temporary AWS access credentials from FAWS. For convenience, these can be set in environment variables.
+- `--environments` an environments.yml file that defines the different environments (or stages) and the associated AWS account for each. The format is as described in the `fleece run` command.
+
+The config commands work with two types of config files. The `config.yml` file is a "closed" config file, where all sensitive values are encrypted. Developers typically do not edit this file but instead export it to an "open" configuration file where sensitive variables appear in plain text for editing. As soon as changes are made, the open config file is imported, which re-encrypts everything and writes an updated `config.yml`.
+
+The open configuration format is as follows:
+
+```
+keys:                                 # kms encryption keys for each environment
+  dev: dev-key-here
+  prod: prod-key-here
+config:
+  foo: bar                            # plain text variable
+  password:                           # per-stage values, encrypted
+    -dev: :encrypt:my-dev-password
+    -prod: :encrypt:my-prod-password
+  nested:                             # nested dictionaries
```

Comment:
If you need values for a custom stage that are different to any of the official stages, then you need to add your custom stage to `environments.yml`, define a key for this stage, and then add your encrypted value here as an additional line.

If, on the other side, you want your custom stage to inherit one of the official stages, then when you deploy, you can specify the parent stage after a colon, e.g. `miguelgrinberg:dev`. So then the `miguelgrinberg` stage will get any configuration values that are not explicitly set for that stage from `dev`.

Reply:
The point of custom stages is that they are created ad-hoc, exist for each developer, and shouldn't require extra configuration (when a new developer starts contributing). I think it's nice that you can define a fallback stage, so that would be enough for our use case, but I can't see this feature documented anywhere.